src/main/webapp/css/user-page.css

```diff
@@ -13,6 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+body {
+  background-color: #CCFFFF;
```

Comment:
This should be in main.css so that it applies to all pages. I think you should also remove the bgcolor from the body in index.html so that this color gets applied consistently on all pages.